### PR TITLE
Load audio from local server

### PR DESCRIPTION
**Pull request by NgJason**
## Description
Load audio from local server. Updated ref link.

## Motivation and Context
Cleaner code.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by chuyskywalker** [comment]
I've seen some other changes where people wanted the links to be relative instead of absolute, just FYI.

File: static/map.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ var map_pokestops = {}
 var map_lure_pokemons = {}
 var map_scanned = {}
 var gym_types = ["Uncontested", "Mystic", "Valor", "Instinct"];
-var audio = new Audio('https://github.com/AHAAAAAAA/PokemonGo-Map/raw/develop/static/sounds/ding.mp3');
+var audio = new Audio('/static/sounds/ding.mp3');
```
Comment (MonkeyLeeT) [inline static/map.js]: "static/sounds/ding.mp3", taking out the first slash will work better imo.

**Comment by NgJason** [comment]
@chuyskywalker @MonkeyLeeT thanks!

**Comment by MonkeyLeeT** [comment]
+1. Tested working well.

**Comment by andrewda** [comment]
👍

**Pull request merged**